sample-game-common/src/main/java/com/squareup/sample/tictactoe/RunGameReactor.kt

```diff
@@ -126,6 +123,8 @@ class RunGameReactor(
         }
 
         onEvent<PlayAgain> {
+          workflows.abandonWorker(logGameWorker)
```

Comment:
We should have a comment here saying why this is necessary, and what will happen if we don't.